src/app/page.tsx

```diff
@@ -77,7 +77,7 @@ export default function Home({
         </div>
       ) : (
         <div className="flex-grow">
-          <DevToolsUI diagnosticData={diagnosticData} />
+          <DevToolsUI diagnosticData={diagnosticData} slackFileId={fileId} />
```

Comment:
Nit: I'd name the fileID more generically since we're looking at eventually moving away from slack as the place to host bug report files

Reply:
yup, I think it might make sense to change this once we purge debugger from all mentions of slack